activities/tool-management/odoo.md

```diff
@@ -16,8 +16,6 @@ We offer training to new staff.
 
 ## Odoo management
 
-Odoo Community is currently hosted and maintained by a service provider in the UK. We are migrating our instance to Openworx in November 2020.
```

Comment:
Would be good to still mention here who our service provider is. Agree that the migration stuff can all be moved, but it is confusing as service provider is mentioned in the next line, but then it is unclear who/what it is referring to.